sdk/lib/atl/atlbase.h

```diff
@@ -1074,11 +1074,11 @@ class CRegKey
         ATLASSERT(lpszKeyName);
 
         HKEY hKey = NULL;
-        LONG lRes = ::RegOpenKeyEx(hKeyParent, lpszKeyName, NULL, samDesired, &hKey);
+        LONG lRes = ::RegOpenKeyEx(hKeyParent, lpszKeyName, 0, samDesired, &hKey);
```

Comment:
Why?

Reply:
The 3rd argument of `RegOpenKeyEx` is integer type. `NULL` for it makes warnings.